src/inverseRobot.js

```diff
@@ -23,8 +23,18 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
-}
+  const object = {};
 
+  for (const key in robot) {
+    object[robot[key]] = key;
+  }
```

Comment:
You could check in your loop is your object already `hasOwnProperty(robot[key])` and return null in this case. This way you code could have less iterations.